roles/postgresql/tasks/main.yml

```diff
@@ -58,29 +46,9 @@
   register: "postgresql_image_pulled"
   tags: ["prepare", "prepare-postgresql"]
 
-- name: "Set container version label to version of pulled image"
```

Comment:
The version label defaults to `postgresql_container_version`, so without this, only the major version will be in the label. I'm therefore in favor of keeping this.